packages/apps-config/src/api/spec/subspace.ts

```diff
@@ -1,111 +1,133 @@
 // Copyright 2017-2021 @polkadot/apps-config authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
-import type { OverrideBundleDefinition } from '@polkadot/types/types';
-
 // structs need to be in order
 /* eslint-disable sort-keys */
 
+import type { Observable } from 'rxjs';
+import type { ApiInterfaceRx } from '@polkadot/api/types';
+import type { Bytes, Struct, U8aFixed, u64 } from '@polkadot/types';
+import type { OverrideBundleDefinition, Registry } from '@polkadot/types/types';
+
+import { combineLatest, map } from 'rxjs';
+
+import { bestNumber, bestNumberFinalized, bestNumberLag, getBlock, subscribeNewBlocks } from '@polkadot/api-derive/chain';
+import { memo } from '@polkadot/api-derive/util';
+import { AccountId, Digest, Header } from '@polkadot/types/interfaces';
+
+interface HeaderExtended extends Header {
+  readonly author: AccountId | undefined;
+}
+
+interface Solution extends Struct {
+  readonly publicKey: AccountId;
+  readonly nonce: u64;
+  readonly encoding: Bytes;
+  readonly signature: Bytes;
+  readonly tag: U8aFixed;
+}
+
+interface SUBPreDigest extends Struct {
```

Comment:
Here and in other places let's use `Sub` instead of `SUB` in interfaces and such:
```suggestion
interface SubPreDigest extends Struct {
```